### PR TITLE
dragonball: output virtio-net device metrics to runtime

### PR DESCRIPTION
**Pull request by lisongqian**
Virtio-net device manager adds net device metrics to METRICS when a device is created and remove metrics when a device is dropped.

Fixes: #7248

Signed-off-by: lisongqian <mail@lisongqian.cn>

### CODE REVIEW

**Comment by lisongqian** [comment]
Block on #7505.

File: src/dragonball/src/metric.rs [hunk 1]
```diff
@@ -5,12 +5,15 @@
 use dbs_utils::metric::SharedIncMetric;
 use lazy_static::lazy_static;
 use serde::Serialize;
+use std::collections::HashMap;
```
Comment (ZizhengBian) [inline src/dragonball/src/metric.rs:8]: refactor use statement

**Comment by lisongqian** [comment]
This PR will work on #8392.

**Review by ZizhengBian — APPROVED** [review]
LGTM

**Pull request closed without merging**